### DISCUSSION TITLE
正在刷新订阅，请稍后操作

### DISCUSSION THREAD

**Discussion by suixin0230** [gkd-kit/gkd#822] [General]
是bug吗？还是什么原因![Screenshot_20241215_135857_li.songe.gkd.jpg](https://github.com/user-attachments/assets/42d409f6-a799-487c-b5f6-e940ce906456)
下滑刷新不了

**Comment by lisonge** [comment]
结束应用后重启

**Reply by suixin0230** [reply]
好了感谢🙏🙏🙏
